Assets/Scripts/Controllers/PlayerInputController.cs

```diff
@@ -18,4 +18,9 @@ public override bool RetrieveJumpButtonReleased()
     {
         return MInput.GetKeyUp(KeyCode.Space) || MInput.GetPadUp(GamepadCode.ButtonSouth);
     }
+
+    public override bool RetrieveDashInput()
+    {
+        return MInput.GetKeyDown(KeyCode.LeftShift);
```

Comment:
Could we add gamepad button as well? (Can also happen in a later PR too) I'm thinking left bumper would be a good dash button, or we can use RTrigger if we want to copy HK.

Reply:
TIL there's a difference between bumper and trigger